### PR TITLE
sustainability blogpost

### PR DESCRIPTION
**Pull request by mcbloch**

### CODE REVIEW

File: content/blog/19-20/sustainability_blog.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Het plan
+
+We willen met Zeus een actieve werkgroep oprichten rond duurzaamheid waarmee we af en toe samen komen. Het leek ons een goed idee om onze leden actief mee te betrekken in deze conversatie en komen daarom met dit voorstel naar jullie.
+
+Het is onze bedoeling een werkgroep van ongeveer 7 personen samen te stellen: 5 normale leden en 2 bestuursleden. Deze werkgroep zou actief instaan voor het bespreken van voorstellen omtrent duurzaamheid binnen de werking van Zeus maar kan ook projecten en activiteiten organiseren en begeleiden. Er zou nauwe samenwerking zijn met het bestuur voor belangrijke beslissingen maar met deze werkgroep hopen we de discussie meer open te stellen en meer interessante ideeën binnen te trekken.
+
+## Interesse?
+
+Aan iedereen die geïnteresseerd is om hierin mee te werken, let us know! Dan stellen wij een team samen en kunnen we erin vliegen. Het idee zou zijn een eerste samenkomst te hebben na de start van het nieuwe semester in Februari.
```
Comment (niknetniko) [inline content/blog/19-20/sustainability_blog.md]:
```suggestion
Aan iedereen die geïnteresseerd is om hieraan mee te werken, let us know! Dan stellen wij een team samen en kunnen we erin vliegen. Het idee zou zijn een eerste samenkomst te hebben na de start van het nieuwe semester in februari.
```

**Comment by redfast00** [comment]
@mcbloch ging je dan nog wat meer info geven over ons teamtrees dink?

**Comment by mcbloch** [comment]
Working on it.

File: content/blog/19-20/sustainability_blog.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+We hebben als startschot van de beweging binnen Zeus een organisatie uitgezocht die actie onderneemt ten voordele van het klimaat en hebben uiteindelijk voor [TeamTreas](teamtrees) gekozen. 
+Dit is een samenwerking tussen de bekende youtuber Mr. Beast en de [Arbor Day Foundation](arborday). Deze laatste zet zich dagelijks in om bossen te planten en te herstellen.
+
+Op ons laatste evenement van 2019 hebben we met alle Zeus-leden genoeg geld verzameld om 475 bomen te planten voor TeamTrees (1$ = 1🌲)! 
+We hebben hiermee onze eigen verwachtingen zeker overtroffen en willen nogmaals iedereen bedanken die een steentje heeft bijgedragen. Elk klein bedrag heeft geholpen. 
+
+
+Na deze schitterende start is het misschien het moment om stil te staan bij wat we kunnen doen als vereniging.
+Er zijn vele manieren om een bewuster om te gaan en een bijdrage te leveren aan ons klimaat zoals letten op onze afvalproductie en ons energieverbruik.
```
Comment (niknetniko) [inline content/blog/19-20/sustainability_blog.md]:
```suggestion
Er zijn vele manieren om een bewuster om te gaan en een bijdrage te leveren aan ons klimaat, zoals letten op onze afvalproductie en ons energieverbruik.
```

File: content/blog/19-20/sustainability_blog.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Maar hier moet het zeker niet bij blijven. Zeus heeft zijn eigen unieke sterktes en specialisaties, waar zeker veel potentieel in zit. 
+Waarom zouden we dit dan niet proberen benutten?
```
Comment (niknetniko) [inline content/blog/19-20/sustainability_blog.md]:
```suggestion
Waarom zouden we dit dan niet proberen te benutten?
```

File: content/blog/19-20/sustainability_blog.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Het plan
+
+We willen met Zeus een actieve werkgroep oprichten rond duurzaamheid waarmee we af en toe samen komen. 
+Het leek ons een goed idee om onze leden actief mee te betrekken in deze conversatie en komen daarom met dit voorstel naar jullie.
+
+Het is onze bedoeling een werkgroep van ongeveer 7 personen samen te stellen: 5 normale leden en 2 bestuursleden. 
+Deze werkgroep zou actief instaan voor het bespreken van voorstellen omtrent duurzaamheid binnen de werking van Zeus maar kan ook projecten en activiteiten organiseren en begeleiden. 
+Er zou nauwe samenwerking zijn met het bestuur voor belangrijke beslissingen maar met deze werkgroep hopen we de discussie meer open te stellen en meer interessante ideeën binnen te trekken.
```
Comment (niknetniko) [inline content/blog/19-20/sustainability_blog.md]:
```suggestion
Er zal nauw samengewerkt worden met het bestuur voor belangrijke beslissingen, maar met deze werkgroep hopen we de discussie meer open te stellen en meer interessante ideeën binnen te trekken.
```
(not really a request, more of an optional suggestion)

File: content/blog/19-20/sustainability_blog.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+## Interesse?
+
+Aan iedereen die geïnteresseerd is om hieraan mee te werken, let us know! Dan stellen wij een team samen en kunnen we erin vliegen. 
+Het idee zou zijn een eerste samenkomst te hebben na de start van het nieuwe semester in februari.
+
+Wil je je niet kandidaat stellen voor de core van de werkgroep maar heb je wel een origineel idee of een suggestie, 
+  we zien je graag komen op ons chatplatform in het kanaal voor duurzaamheid [~sustainability][sustainmm].
+
+Momenteel ben ik (flynn op Mattermost) de lead van het project maar bij vragen kunt je altijd terecht bij je favoriete of meest bereikbare bestuurslid. 
+De algemene bestuurschat vind je hier [~bestuur][bestuurmm].
```
Comment (niknetniko) [inline content/blog/19-20/sustainability_blog.md]:
```suggestion
De algemene bestuurschat vind je in [~bestuur][bestuurmm].
```

**Comment by mcbloch** [comment]
@niknetniko Thanks for the suggestions. I applied them!

**Comment by werthen** [comment]
Merge?

**Pull request merged**